### PR TITLE
feat(discover-quick-context) Events design changes.

### PR DESCRIPTION
**Pull request by Abdkhan14**
1. Added copyable event id to context header.
2. Added title with stacktrace for errors, if title not already in the results table.
3. Updated tests.

### CODE REVIEW

File: static/app/views/eventsV2/table/quickContext.spec.tsx [hunk 1]
```diff
@@ -589,6 +591,25 @@ describe('Quick Context', function () {
...
     });
 
+    it('Renders event id header', async () => {
+      jest.spyOn(ConfigStore, 'get').mockImplementation(() => null);
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/events/sentry:6b43e285de834ec5b5fe30d62d549b20/',
+        body: makeEvent({type: EventOrGroupType.ERROR, entries: []}),
+      });
+
+      delete defaultRow.title;
```
Comment (narsaynorath) [inline static/app/views/eventsV2/table/quickContext.spec.tsx:601]: Is this delete necessary for the test? I assume you may have added it to recreate the state where we have the "+" icon in the title, but I don't see that being tested here

File: static/app/views/eventsV2/table/quickContext.tsx [hunk 2]
```diff
@@ -93,9 +90,26 @@ function getHoverBody(
...
 // NOTE: Will be adding switch cases as more contexts require headers.
 function getHoverHeader(dataRow: EventData, contextType: ContextType) {
-  return isReleaseContext(contextType) ? (
-    <VersionHoverHeader releaseVersion={dataRow.release} />
-  ) : null;
+  switch (contextType) {
+    case ContextType.RELEASE:
+      return (
+        <HoverHeader
+          title="Release"
```
Comment (narsaynorath) [inline static/app/views/eventsV2/table/quickContext.tsx:97]: This and 105 need `t()`
```suggestion
          title={t('Release')}
```

**Review by narsaynorath — APPROVED** [review]

File: static/app/views/eventsV2/table/quickContext.spec.tsx (hunk 1 above)
Comment (Abdkhan14) [reply static/app/views/eventsV2/table/quickContext.spec.tsx:601]: Yeah, don't need that there.

File: static/app/views/eventsV2/table/quickContext.tsx (hunk 2 above)
Comment (Abdkhan14) [reply static/app/views/eventsV2/table/quickContext.tsx:97]: good catch!

**Pull request merged**